### ISSUE TITLE
Redirect from login page if user logged

### ISSUE THREAD

**Issue by pbujok** [IdentityServer/IdentityServer3#1596]
My question is related to an issue which we are investigating. If you open two (or more) login pages and you have different nonces, then after succesful sign in, every following logging attempt will lead to changing the session state and in fact to signing out from every aplication. Is there a good way around that? We are considering a flow similar to facebook - if user was on a login page, and was signed in by another tab, then we could redirect him to the appropriate url.

**Comment by brockallen** [comment]
Perhaps it makes sense to check if the user is already logged in before we issue a new session state value. @leastprivilege and I will discuss.

**Comment by pbujok** [comment]
We carefully read part of OpenId standard, and compared it to your sample. 

> When the RP detects a session state change, it SHOULD first try a prompt=none request within an iframe to obtain a new ID Token

We discovered that you had already implemented solution for our issue. We uncommented your code and this way we don't have problem with signing out after login if you are logged.
<pre>mgr.renewTokenSilentAsync().then(function () { 
    console.log("renewTokenSilentAsync success");
}, function () {
    console.log("renewTokenSilentAsync failed");
});
</pre>
(https://github.com/IdentityServer/IdentityServer3.Samples/tree/master/source/Clients/JavaScriptImplicitClient)
